demos/collection/grid.php

```diff
@@ -69,5 +69,12 @@
     [$sel->jsChecked()]
 ));
 
+$grid->addDecorator($model->titleField, [\Atk4\Ui\Table\Column\Link::class, 'url' => 'https://hmpg.net']);
+
+
+$grid->table->onRowClick(function() {
```

Comment:
needs test

also test the checkbox, as with link, it is hard to test if the event was propagated or not when navigated (URL changed)